lib/ansible/modules/utilities/logic/include_vars.py

```diff
@@ -58,7 +58,12 @@
     description:
       - List of file extensions to read when using C(dir).
     default: ['yaml', 'yml', 'json']
-    required: False
+    required: false
+  override:
```

Comment:
To avoid confusion this should match the 'global setting' ```hash_behaviour = replace|merge```, also we are in freeze for 2.3, so this will probably be merged for 2.4